### PR TITLE
Update Gradle wrapper to 5.6

### PR DESCRIPTION
**Pull request by jianglai**
This change is [Reviewable](https://reviewable.io/reviews/google/nomulus/232)

### CODE REVIEW

**Comment by gbrodman** [comment]
It seems like there are some issues with the Gradle upgrade and the shadow jar plugin that we use to build the Nomulus tool.

**Comment by jianglai** [comment]
This is due to https://github.com/johnrengelman/shadow/issues/508. I'll push an update soon.

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
+org.gradle.jvmargs=-Xmx1024m
```
Comment (gbrodman) [inline gradle.properties]: Isn't this the default? Is specifying it necessary?

Comment (jianglai) [reply gradle.properties]: Somehow it was giving me OOM error when I didn't specify this...

Comment (gbrodman) [reply gradle.properties]: I was apparently wrong, or the posts online I found were wrong. Default is 512 MB apparently 

https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory

**Pull request merged**